src/ros/ros2/ros2-monitor.ts

```diff
@@ -0,0 +1,128 @@
+// Copyright (c) Andrew Short. All rights reserved.
```

Comment:
nit: please consider using Microsoft copyright header

Reply:
Good catch. Fixed it.